### DISCUSSION TITLE
您设置的间隔复习的思路是？

### DISCUSSION THREAD

**Discussion by vipyanzg** [oldwinter/knowledge-garden#50] [提问]
实不相瞒，我在倒腾插件的时候，破坏了间隔复习，以至于在homeoage中点击 canlendar→anki 回顾笔记时，报错了，打不开

**Comment by oldwinter** [comment]
那个是一个用andvanced uri实现的，可能跟vault名字绑定，你可以直接点右下角开始笔记复习，或者cmd➕p命令找到spaced 相关命令
